### ISSUE TITLE
__dirname is not defined

### ISSUE THREAD

**Issue by towfiqi** [tomayac/esm-potrace-wasm#3]
Just imported the package to use it in an existing nextjs project and got this error:
![Error Details](https://i.imgur.com/fvHZ7cI.png)

**Comment by tomayac** [comment]
Could this be a Next.js thing? Some searching around suggests, declaring the `ROOT` fixes it.

```js
// inside next.config.js
module.exports = {
  env: {
    ROOT: __dirname,
  }
}
```

If this doesn't work, please comment and I will re-open the issue.

**Comment by towfiqi** [comment]
@tomayac Thanks for looking into this. Just tied your suggestion and still getting the same error.

**Comment by tomayac** [comment]
OK, I have traced this down. Right now my Wasm build is a bit of a mess and I am not using the "proper" `-s "MODULARIZE=1" -s "EXPORT_ES6=1"` option, but instead  shoehorn the module part:

https://github.com/tomayac/esm-potrace-wasm/blob/85161431c54d9ae4d931e7388c3954dc28c54fb2/package.json#L19

If you know how to continue successfully what I have started in #2, this would be super helpful.

Right now, I fear this doesn't work in Node.js in the current shape. Maybe give [this fork](https://github.com/nohtcoltd/potrace-wasm) a shot meanwhile.
